GOVERNANCE.md

```diff
@@ -0,0 +1,35 @@
+# OSPOlogy Repo Governance
+
+This document defines the governance process for the OSPOlogy repo. This doc outlines the various responsibilities of contributor roles in OSPOlogy, and clarifies how people can make their way up the OSPOlogy contributor ladder.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| OSPOlogy Contributor | Active contributor in the community | One successful merge of significant pull requests within the OSPOlogy repo | [OSPOlogy contributors ](https://github.com/orgs/todogroup/teams/ospology-contributors) GH team|
+| OSPOlogy Maintainer | See "Expectations from Maintainers" bellow| <ul><li>5-10 successful merges of significant pull requests in two or more initiatives within the repo</li><li>Fulfill the "Expectations from Maintainers"</li><li>2/3 majority thumbs up on the PR within settings.yml from existing maintainers.</li></ul> | [settings.yml file](https://github.com/todogroup/ospology/blob/main/.github/settings.yml#L18)|
+
+## Changes in Maintainership
+
+5-10 successful merges of significant pull requests in two or more initiatives within the repo within the OSPOlogy repo, any current maintainer can reach the author behind the pull requests and ask them if they are willing to become a maintainer. They can also send a PR to [settings.yml file](https://github.com/todogroup/ospology/blob/main/.github/settings.yml#L18) and request the existing maintainers to vote them in. 2/3 majority thumbs up on the PR within settings.yml from existing maintainers will be required for a person to become an OSPOlogy maintainer.
+
+If a maintainer feels they can not fulfill the "Expectations from Maintainers", they are free to step down.
+
+Existing maintainers can remove maintainers who become inactive or disruptive on a 2/3 majority vote from existing maintainers. If people step down or are removed, they will be added to the OSPOlogy **emeritus list (TBD)**
```

Comment:
Instead of disruptive, maybe "... become inactive or are not acting in the best interests of the TODO Group on a 2/3 ..."